wesnoth-ei.po

```diff
@@ -8565,19 +8579,23 @@ msgid ""
 "... Owaec... why do you fight me? Join us... ride alongside your father... "
 "defend the plains..."
 msgstr ""
+"... Owaecu…, proč se mnou bojuješ? Přidej se k nám..., jeď po boku svého "
```

Comment:
```suggestion
"... Owaecu..., proč se mnou bojuješ? Přidej se k nám..., jeď po boku svého "
```

Reply:
Netuším, proč mi to občas hodí tu divnou trojtečku, když většinou je to v pořádku.